src/custom/utils/appZi.ts

```diff
@@ -0,0 +1,69 @@
+import ReactAppzi from 'react-appzi'
+
+export const isAppziEnabled =
+  process.env.REACT_APP_FEEDBACK_ENABLED_DEV === 'true' || process.env.NODE_ENV === 'production'
+
+export const FEEDBACK_KEY = process.env.REACT_APP_APPZI_FEEDBACK_KEY || 'f7591eca-72f7-4888-b15f-e7ff5fcd60cd'
+export const NPS_KEY = process.env.REACT_APP_APPZI_FEEDBACK_KEY || '55872789-593b-4c6c-9e49-9b5c7693e90a'
+
+const APIZI_TOKEN = process.env.REACT_APP_APPZI_TOKEN || '5ju0G'
+
+// const DEFAULT_SETTINGS: AppziCustomSettings = {
+//   tradeCount: 10,
+//   userJustTraded: 'true',
+// }
+
+declare global {
+  interface Window {
+    appzi?: {
+      openWidget: (key: string) => void
+    }
+    appziSettings: {
+      data: AppziCustomSettings
+    }
+  }
+}
+
+interface AppziCustomSettings {
+  tradeCount: number
+  userJustTraded: 'true' | 'false'
```

Comment:
I think it can be a boolean. I was trying out thinks cause at first was not working. But was not working because of some params in appZi side, so now I think u can use:

```
window.appziSettings = {
    data: {
      'userTradedOrWaitedForLong': true,
    }
  }
```